Documentation/README_HandCoach.md

```diff
@@ -0,0 +1,76 @@
+# Hand Coach
+![](../Documentation/Images/HandCoach/MRTK_UX_HandCoach_Main.jpg)
+
+Hand Coach is 3D modeled hand which is triggered when the system does not detect the user’s hands. This is implemented as a “teaching” component that helps guide the user when the gesture has not been taught. If users have not done the specified gesture for a period, the hands will loop with a delay. Hand Coach could be used to represent pressing a button or picking up a hologram.   
+
+
+The current interaction model represents a wide variety of gesture controls such as scrolling, far select, and near tap. Below is a full list of existing JIT hands: 
+
+- Near Tap – Used for buttons or close interactable objects 
```

Comment:
Near Tap, Palm up and Scroll all appear to be using " – " between title and discription, whereas the rest of the list is using "- ". We should probably pick one style and be consistent in this list. I think space/en dash/space looks neater than dash/space.

Reply:
fixed